package/src/components/ImageGallery/components/ImageGalleryFooter.tsx

```diff
@@ -204,7 +207,11 @@ export const ImageGalleryFooterWithContext = <
           {leftElement ? (
             leftElement({ openGridView, photo, share, shareMenuOpen })
           ) : (
-            <TouchableOpacity disabled={shareMenuOpen} onPress={share}>
+            <TouchableOpacity
+              accessibilityLabel='share-button'
```

Comment:
This label should be a human readable label.
https://reactnative.dev/docs/accessibility#accessibilitylabel

> When a view is marked as accessible, it is a good practice to set an accessibilityLabel on the view, so that people who use VoiceOver know what element they have selected. VoiceOver will read this string when a user selects the associated element.